c10/util/signal_handler.cpp

```diff
@@ -37,6 +37,7 @@ std::atomic<int> sighupCount(0);
 std::atomic<int> hookedUpCount(0);
 
 void handleSignal(int signal) {
+  // NOLINTNEXTLINE(bugprone-switch-missing-default-case)
```

Comment:
Will this error if it handles an unknown signal type? Is that what we want?

Reply:
This file should be rarely changed. If some cases were missed , it should be easy to detected in runtime